ThermalTalk/Common/BasePrinter.cs

```diff
@@ -360,28 +364,27 @@ public virtual ReturnCode PrintDocument(IDocument doc)
                     SetFont(sec.Font),
 
                     // Send the actual content
-                    internalSend(sec.GetContentBuffer(doc.CodePage)),
+                    AppendToDocBuffer(sec.GetContentBuffer(doc.CodePage)),
                 };
-                
+
 
                 if (sec.AutoNewline)
                 {
-                    subResults.Add(PrintNewline());
+                    PrintNewline();
                 }
 
                 // Remove effects for this section
-                subResults.Add(RemoveEffect(sec.Effects));
-                
-                results.AddRange(subResults);
+                RemoveEffect(sec.Effects);
             }
 
             // Undo all the settings we just set
-            results.Add(SetJustification(oldJustification));
-            results.Add(SetScalars(oldWidth, oldHeight));
-            results.Add(SetFont(oldFont));
+            SetJustification(oldJustification);
+            SetScalars(oldWidth, oldHeight);
+            SetFont(oldFont);
 
-            return results.Select(x => x != ReturnCode.Success).Any()
-                ? ReturnCode.ExecutionFailure : ReturnCode.Success;
+            // Transfer document to printer
+            var result = FormFeed();
```

Comment:
This is why we have the difference in behavior. I wasn't sure what would be less surprising.

1. Not actually printing when PrintDocument is called
2. Changing the process to always require an explicit call to form feed

We can bump the rev to 2.0, update the docs, and add a big change notice to the readme.

Reply:
> We can bump the rev to 2.0, update the docs, and add a big change notice to the readme.

That works for me.